core/Dockerfile

```diff
@@ -64,6 +64,9 @@ EXPOSE 9000/tcp
 EXPOSE 8812/tcp
 EXPOSE 9009/tcp
 
+RUN ulimit -S unlimited
+RUN ulimit -H unlimited
```

Comment:
@mariusgheorghies does this have any effect?

Reply:
this only applies to the current session, to have the effect you are looking for, it needs to be added at runtime with `docker run  --ulimit`. See https://docs.docker.com/engine/reference/commandline/run/#ulimit

Reply:
@marregui can you remove these lines, it give false impression